app/form_models/jobseekers/job_application/review_form.rb

```diff
@@ -0,0 +1,15 @@
+class Jobseekers::JobApplication::ReviewForm
+  include ActiveModel::Model
+
+  attr_accessor :confirm_data_accurate, :confirm_data_usage, :completed_steps
+
+  validates :confirm_data_accurate, acceptance: true
+  validates :confirm_data_usage, acceptance: true
+  validate :all_steps_completed
+
+  def all_steps_completed
+    return if JobApplication.completed_steps.keys.map { |step| completed_steps.include?(step) }.all?
```

Comment:
Is this equivalent to the terser
```ruby
JobApplication.completed_steps.keys.all? { |step| completed_steps.include?(step) }
```
?

Reply:
Nice, didn't know `all?` could have a block!